api/app/signals/apps/api/v1/serializers/expression.py

```diff
@@ -0,0 +1,47 @@
+from datapunt_api.serializers import HALSerializer
+from rest_framework import serializers
+from rest_framework.exceptions import ValidationError
+
+from signals.apps.signals.models import Expression, ExpressionContext, ExpressionType
+
+
+class ExpressionContextSerializer(serializers.ModelSerializer):
+    type = serializers.SerializerMethodField('get_type_name')
+
+    class Meta:
+        model = ExpressionContext
+        fields = ('identifier', 'identifier_type', 'type',)
+
+    def get_type_name(self, obj):
```

Comment:
See comment above when creating the type

Reply:
check